build/common.go

```diff
@@ -130,6 +130,11 @@ func (Build) Linux() error {
 	return buildBackend(newBuildConfig("linux", "amd64"))
 }
 
+// LinuxArm64 builds the back-end plugin for Linux on arm64.
+func (Build) LinuxArm64() error {
```

Comment:
Acronyms should have [consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) in Go:
```suggestion
// LinuxARM64 builds the back-end plugin for Linux on ARM64.
func (Build) LinuxARM64() error {
```